### PR TITLE
chore(toolchain): Update readme with docker native client step

### PR DESCRIPTION
**Pull request by phil-linnell**
@arnau Please could you take a look at this and just check to see if it makes sense to you?

### CODE REVIEW

File: docs/DOCKER-MACHINE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Docker Machine
+
+Before the Docker native client we used docker-machine. Here are the steps for this original setup process.
+
+## Docker dependencies
+
+The project is managed via Docker containers. Right now we're using Docker
+version 1.9.0 with Docker Machine version 0.5.0.
```
Comment (arnau) [inline docs/DOCKER-MACHINE.md]: The current versions are Docker 1.12 and Docker Machine 0.8

File: docs/DOCKER-MACHINE.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Commands below assume OS X and preference to install libraries via Homebrew.
```
Comment (arnau) [inline docs/DOCKER-MACHINE.md]: I think it's better to say that the best way to install any of these Docker tools it to follow the current Docker documentation on https://docker.github.io/engine/installation/

On top of that, installing Docker native installs Docker Machine as well so they're not exclusive approaches.  A link to the source seems more robust to me.

Comment (phil-linnell) [inline docs/DOCKER-MACHINE.md]: So maybe I should drop this legacy setup process altogether?

Comment (arnau) [inline docs/DOCKER-MACHINE.md]: I think it's alright to do so.  Perhaps you want to check with @andydutch if anyone in his side is not using Docker native.

Comment (phil-linnell) [inline docs/DOCKER-MACHINE.md]: Good thinking. @andydutch I'm making some modifications to the setup process in README.md. Thinking about killing the old setup and just replacing with essentially "Download Docker Native Client". Any thoughts about keeping the legacy setup process doc?

**Pull request closed without merging**